src/cim/components/Agent/tableUtils.tsx

```diff
@@ -163,12 +163,16 @@ export const infraEnvColumn = (
     cell: (host) => {
       const agent = agents.find((a) => a.metadata?.uid === host.id) as AgentK8sResource;
       const infraEnvName = getInfraEnvNameOfAgent(agent);
-
       let title: React.ReactNode = infraEnvName || 'N/A';
```

Comment:
Actually a little better would be

```suggestion
      const title = infraEnvName ?  <Link
            to={`/multicloud/infrastructure/environments/details/${agent.metadata?.namespace}/${infraEnvName}/overview`}
          >
            {infraEnvName}
          </Link> : 'N/A';
```

sorry for the mess :)

You should also remove `getInfraEnvLink` and `getInfraEnvLinkType` from function args and every reference within the repo

Reply:
No problem. I've made the changes